src/sidebar/main.js

```diff
@@ -270,6 +270,54 @@ function getDebuggerInstance (selectedDomElement, domDocument) {
 		return this._catberry.version || '';
 	};
 
+	/**
+	 * Gets check.
+	 * @returns {Array}
+	 */
+	CatberryDebugger.prototype.getActiveCheck = function () {
```

Comment:
I can not use the name `getComponentsWithNotValidId` because on render function eval this code `'.getActive' + methodNamePart + '()'`

I change 'check' to 'ids'